components/AddForm.js

```diff
@@ -0,0 +1,70 @@
+import CheckIcon from '../public/assets/check.svg';
+import styled from 'styled-components';
+
+export default function AddForm({ onAddItem, setAddActive }) {
+  function handleSubmit(event) {
+    event.preventDefault();
+    const form = event.target.elements;
+    const data = {
+      name: form.name.value,
+      date: form.date.value,
+      quantity: form.quantity.value,
```

Comment:
```suggestion
      quantity: form.quantity.value,
      id: nanoid(),
```

Its probably best to already add an ID here, for example with nano ID, this could save you some trouble later :)